### PR TITLE
Fix rendering artifacts

### PR DESCRIPTION
**Pull request by guilhermefelipecgs**
Issue #14390

### CODE REVIEW

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -270,7 +270,7 @@ void EditorNode::_notification(int p_what) {
 
 		Engine::get_singleton()->set_editor_hint(true);
 
-		get_tree()->get_root()->set_usage(Viewport::USAGE_2D_NO_SAMPLING); //reduce memory usage
+		get_tree()->get_root()->set_usage(Viewport::USAGE_2D); //reduce memory usage
```
Comment (akien-mga) [inline editor/editor_node.cpp]: The comment is no longer relevant..

**Review by reduz — COMMENTED** [review]
Its really odd that this causes the issue, will need to check. As maybe a gl id is being deleted  when it shouldn't

File: editor/editor_node.cpp (hunk 1 above)
Comment (guilhermefelipecgs) [reply editor/editor_node.cpp]: Comment removed.

**Comment by guilhermefelipecgs** [comment]
ok

**Comment by hpvb** [comment]
@guilhermefelipecgs did these rendering artifacts show up recently?

**Comment by hpvb** [comment]
Never mind, I missed the issue link. Sorry for the noise.

**Comment by reduz** [comment]
ah, it was not closed automatically

**Pull request closed without merging**